Dockerfile

```diff
@@ -3,6 +3,10 @@
 # https://github.com/docker-library/repo-info/tree/master/repos/alpine/remote
 FROM alpine@sha256:bc41182d7ef5ffc53a40b044e725193bc10142a1243f395ee852a8d9730fc2ad
 ARG EXTENSION_FILE
+ARG BUILD_DATE
 COPY ${EXTENSION_FILE} /opt/elastic-apm-extension
-COPY /NOTICE.txt /opt/NOTICE.txt
-COPY /dependencies.asciidoc /opt/dependencies.asciidoc
+COPY NOTICE.txt dependencies.asciidoc /opt/
+
+# Related to reproducible builds
+RUN find /opt -exec touch -t "${BUILD_TIME}" {} \;
```

Comment:
**issue**: did you mean `BUILD_DATE` ?

Reply:
thank you for noticing: https://github.com/elastic/apm-aws-lambda/pull/362/commits/d15a6b9839f1db730b05ded25e0d0a026f0d0636